run-basic-batch-operators-test.sh

```diff
@@ -314,11 +322,14 @@ while IFS=, read -r package_name catalog; do
 	# merge claim.json from each operator to a single csv file
 	./tnf claim show csv -c "$reportDir"/claim.json -n "$package_name" -t "$CNF_TYPE" "$addHeaders" >>"$REPORT_FOLDER"/results.csv
 
+	# extract parser
+	tar -xvf "$reportDir"/*.tar.gz -C "$reportDir"/ results.html
```

Comment:
There is no need for the slash in this context.

Reply:
fixed